app/views/comments/_comment_date.erb

```diff
@@ -4,4 +4,10 @@
       <%= decorated_comment.readable_publish_date %>
     </time>
   </a>
-</div>
+  <% if !decorated_comment.edited_at.blank? %>
+    •
+    <span title="<%= decorated_comment.edited_at.strftime('%A, %d %B %Y, %T') %>">
+      EDITED
+    </span>
+  <% end %>
+</div>
```

Comment:
new line missing

Reply:
new line? Where? Just before the closing div? Or I should add a new line to separate every other line of code?

Reply:
This is what I see when I review the code:

<img width="218" alt="Screenshot 2020-10-02 at 12 39 41 PM" src="https://user-images.githubusercontent.com/146201/94915006-6167a300-04ac-11eb-9109-fc1eeba2f0b2.png">

this is because lines are supposed to end with an empty line, no big deal though :) Usually editors automatically handle this, maybe GitPod doesn't...

Reply:
I actually begin working on this issue in gitpod. I was however having a hard time seeing the changes I pushed from gitpod into my fork on gitpod. So I went back to just using my computer.  I still don't understand the date issue

Reply:
Let me try to explain it from an overview standpoint: when you render a date in Ruby (be it in a `.rb` file or a template `.erb`) what you're essentially saying is: format this date using the server time zone and locale. So the DEV Ruby server takes your instruction and uses the server's locale (English) and timezone (UTC), computes the value of the date and inserts it into the template.

This HTML file is then served to the user's browser. The user though might not be using a computer set to "English and UTC". Mine isn't for example, I live in Italy, which is 1 or 2 hours ahead of UTC, depending on the period of the year. In addition to that, I sometimes keep my browser in my native tongue, which is Italian.

The server knows nothing about this, but the browser does, as it runs on my computer.

So, what JavaScript is able to do is to use this information (again, because it runs on the user's computer) and apply it to such dates so that when the user sees the date they can see it in their preferred locale (in my case Italian) and in their own preferred timezone.

Let me know if any of this isn't clear :)